cri/stream/errors.go

```diff
@@ -0,0 +1,40 @@
+package stream
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+)
+
+// ErrorStreamingDisabled returns error when the streaming method is disabled.
+func ErrorStreamingDisabled(method string) error {
+	return grpc.Errorf(codes.NotFound, fmt.Sprintf("streaming method %s disabled", method))
+}
+
+// ErrorTooManyInFlight returns error when the maximum number of in-flight requests is exceeded.
+func ErrorTooManyInFlight() error {
+	return grpc.Errorf(codes.ResourceExhausted, "maximum number of in-flight requests exceeded")
+}
+
+// WriteError translates a CRI streaming error into an appropriate HTTP response.
+func WriteError(err error, w http.ResponseWriter) error {
```

Comment:
I have no idea that how many grpc code grpc server would return. However, if there are lots of its, maybe we can refer to https://github.com/moby/moby/blob/master/api/server/httputils/errors.go#L98-L129